docs/chapter_searching/binary_search.md

```diff
@@ -337,7 +369,10 @@ $$
 === "JavaScript"
 
     ```js title=""
-
+    // (i + j) 有可能超出 int 的取值范围
+    let m = parseInt((i + j) / 2);
+    // 更换为此写法则不会越界
+    let m = parseInt(i + (j - i) / 2);
```

Comment:
The max value of the JS number is 1.7976931348623157e+308. Therefore, I don't think we have to deal with big-number problem like Java.